src/components/Project/PayProjectForm/PayProjectForm.tsx

```diff
@@ -21,8 +27,10 @@ export function PayProjectForm({ disabled }: { disabled?: boolean }) {
     setPayAmount,
     payInCurrency,
     setPayInCurrency,
-    error,
-    setError,
+    mustBeGreaterThanZeroError,
+    setMustBeGreaterThanZeroError,
+    mustBeLessThanBalanceError,
+    setMustBeLessThanBalanceError,
```

Comment:
**suggestion** now that we're displaying multiple error messages, I think it makes sense to have a single state variable `errorMessage<string>`, and set that to our desired error message. This saves us having more state and makes the jsx logic simpler.

So below, instead of 

```js
return setMustBeGreaterThanZeroError?.(true)
```

we'd have

```js
return setErrorMessage?.('Payment amount can't exceed your wallet balance.')
```

WDYT?